sql/moz-fx-data-marketing-prod/acoustic/contact_raw_v1/metadata.yaml

```diff
@@ -0,0 +1,40 @@
+friendly_name: Contact Database (Acoustic data)
```

Comment:
Unrelated to file but I believe you'll need a `dataset_metadata.yaml` in the `acoustic` dir